packages/teleport/src/services/audit/audit.ts

```diff
@@ -29,8 +29,9 @@ class AuditService {
     const url = cfg.getClusterEventsUrl(clusterId, {
       start,
       end,
-      limit: params.limit ? params.limit : this.maxFetchLimit,
-      startKey: params.startKey ? params.startKey : undefined,
+      limit: this.maxFetchLimit,
+      include: params.filterBy || undefined,
```

Comment:
why `indefined` , we should not be using it as a value in the code

Reply:
it was because of this https://github.com/gravitational/webapps/blob/master/packages/teleport/src/config.ts#L122, these values can be an empty string, and passing in a empty string to `generatePath` will cause an error so it had to be undefined.